src/containers/shared/components/test/Currency.test.tsx

```diff
@@ -16,6 +16,14 @@ describe('Currency', () => {
     wrapper.unmount()
   })
 
+  it('handles currency codes that are 4 characters and include issuer ', () => {
```

Comment:
```suggestion
  it('handles currency codes that are shortened', () => {
```

Reply:
nit - not a blocker lets skip

Reply:
This one should be done - it'll be confusing later otherwise